src/features/auth/loginSlice.jsx

```diff
@@ -0,0 +1,121 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import loginService from "../../services/login.service";
+
+const initialState = {
+  userData: null,
+  loading: false,
+  error: null,
+  isAuthenticated: false,
+  isAdmin: false,
+};
+
+export const makeLogin = createAsyncThunk(
+  "login/auth",
+  async ({ email, password }, { rejectWithValue }) => {
+    try {
+      const response = await loginService.login({ email, password });
+      if (response) {
+        localStorage.setItem("profile", response.data.data.profile);
+        localStorage.setItem("name", response.data.data.name);
+        localStorage.setItem("token", response.data.token);
+        localStorage.setItem("isAdmin", String(response.data.data.isAdmin));
```

Comment:
You can only store token in the localStorage. If you want user info, you will have to fetch them